recipe/views.py

```diff
@@ -1,3 +1,56 @@
-from django.shortcuts import render
+import json
+
+from django.views import View
+from django.http import JsonResponse
+
+from recipe.models import Recipe, RecipeCategory
+
+class RecipeView(View): # 레시피
+    def post(self, request): # 레시피 등록
+        data = json.loads(request.body)
+        
+        try:
+            Recipe.objects.create(
+                name            = data['name'],
+                writer          = data['writer'],
+                image_url       = data['image_url'],
+                content         = data['content'],
+                recipe_category = RecipeCategory(id = data['recipe_category'])
```

Comment:
```suggestion
                recipe_category = RecipeCategory(id = data['recipe_category'])
```
속성을 `recipe_category_id` 로 준다면, `RecipeCategory` 클래스를 사용하지 않고 바로 숫자로 할당하시면 됩니다. 그렇다면 RecipeCategory를 `SELECT`하는 SQL 문을 날리지 않으셔도 되기 때문입니다.

Reply:
넵 수정했습니다.